pkg/services/sqlstore/user.go

```diff
@@ -487,6 +487,31 @@ func DisableUser(cmd *m.DisableUserCommand) error {
 	return err
 }
 
+func BatchDisableUsers(cmd *m.BatchDisableUsersCommand) error {
```

Comment:
This one should be added to the bus though (thanks @markelog for making me think of it).